binding/web/README.md

```diff
@@ -131,33 +140,93 @@ is expressed in [ARPAbet](https://en.wikipedia.org/wiki/ARPABET) phonemes, for e
 - "{read|R IY D} this as {read|R EH D}, please."
 - "I {live|L IH V} in {Sevilla|S EH V IY Y AH}. We have great {live|L AY V} sports!"
 
-### Synthesize Speech
+### Orca Properties
+
+To obtain the complete set of valid characters, call `.validCharacters`. To retrieve the maximum number of
+characters allowed, call `.maxCharacterLimit`. The sample rate of the generated `Int16Array` is `.sampleRate`.
+
+### Usage
+
+#### Streaming Synthesis
+
+To use streaming synthesis, call `streamOpen` to create an `OrcaStream` object.
+
+```typescript
+const OrcaStream = await orca.streamOpen();
+```
+
+Then, call `synthesize` on the `OrcaStream` object to generate speech for a live stream of text:
+
+```typescript
+const textStream = "${TEXT}";
+
+for (const word of textStream.split(" ")) {
+  const pcm = await OrcaStream.synthesize(word + " ");
+  if (pcm !== null) {
+    // handle pcm
+  }
+}
+```
+
+`OrcaStream` buffers input text until there is enough to generate audio. If there is not enough text to generate
+audio, `null` is returned.
+
+When done, call `flush` to synthesize any remaining text, and `close` to delete the `OrcaStream` object.
+
+```typescript
+const flushedPcm = OrcaStream.flush();
+if (flushedPcm !== null) {
+  // handle pcm
+}
+
+OrcaStream.close();
+```
+
+#### Single Synthesis
 
-The `synthesize` function will send the text to the engine and return the speech audio as an `Int16Array`.
+To use single synthesis, simply call `synthesize` directly on the `Orca` instance. The `synthesize` function will send
+the text to the engine and return the speech audio as an `Int16Array`.
 
 ```typescript
 const speechPcm = await orca.synthesize("${TEXT}");
 ```
 
 ### Speech Control
 
-Orca allows for an additional argument to be provided to the `synthesize` method to control the synthesized speech:
+Orca allows for additional arguments to control the synthesized speech.
+These can be provided to the `open_stream` or one of the single mode `synthesize` methods:
 
 - `speechRate`: Controls the speed of the generated speech. Valid values are within [0.7, 1.3]. A higher value produces
   speech that is faster, and a lower value produces speech that is slower. The default value is `1.0`.
 
 ```typescript
 const synthesizeParams = {
-  speechRate: 1.3
+  speechRate: 1.3,
 };
 
+// Streaming synthesis
+const OrcaStream = await orca.streamOpen(synthesizeParams);
+
+// Single synthesis
 const speechPcm = await orca.synthesize("${TEXT}", synthesizeParams);
+
 ```
 
-### Orca Properties
+### Alignment Metadata
 
-To obtain the complete set of valid characters, call `.validCharacters`. To retrieve the maximum number of
-characters allowed, call `.maxCharacterLimit`. The sample rate of Orca is `.sampleRate`.
+Along with the raw PCM or saved audio file, Orca returns metadata for the synthesized audio in single synthesis mode.
+The `OrcaWordAlignment` object has the following properties:
```

Comment:
change `OrcaWordAlignment` to how this object is called in web. Same for `Orca.PhonemeAlignment` etc. below